app/scripts/views/coppa/coppa-age-input.js

```diff
@@ -56,6 +56,22 @@ define(function (require, exports, module) {
       age.val(age.val().substr(0, 3));
     },
 
+    onKeyDown: function (event) {
+      // helper function to check for digit
+      function isKeyADigitOrBackSpace (keyCode) {
+        return ((keyCode === KeyCodes.BACKSPACE) ||
+        (keyCode >= KeyCodes.NUM_0 && keyCode <= KeyCodes.NUM_9) ||
+        (keyCode >= KeyCodes.NUMPAD_0 && keyCode <= KeyCodes.NUMPAD_9));
```

Comment:
I just realized through testing that this no longer allows the arrow keys to be used to move through the text input.